locales/language-icons/fr.yml

```diff
@@ -0,0 +1,4 @@
+1: Je begrijpt of spreekt nog niet veel Nederlands.
+2: Je begrijpt al een beetje Nederlands maar je spreekt nog niet zo goed.
+3: Je begrijpt vrij veel Nederlands en kan ook iets vertellen.
+4: Je begrijpt veel Nederlands en spreek het goed.
```

Comment:
Are these `fr` and `en` translations supposed to be in Dutch?

Reply:
@bertramakers nope, I've added them 👍

Reply:
FYI I didn't approve this one 🙈 
But I didn't have any other comments so it's fine if you fixed it.
We should probably change the settings for this repo to not allow merges without one approval to prevent confusion.